scripts/okify_regtests

```diff
@@ -46,6 +47,53 @@ def artifactory_copy(specfile, dry_run=False):
     subprocess.run(args, check=True)
 
 
+def artifactory_folder_copy(specfile, dry_run=False):
+    """Copy a folder after removing target folder"""
+    jfrog_args = []
+    if dry_run:
+        jfrog_args.append('--dry-run')
+
+    # Since two different jfrog operations are required, need to read in
+    # the spec to perform the delete.
+    with open(specfile) as fh:
+        spec = json.load(fh)
+    pattern = spec['files'][0]['pattern'] + '/'
+    target = spec['files'][0]['target']
+
+    # Remove the target
+    folder = target + Path(pattern).stem
+    args = ['jfrog', 'rt', 'del', folder, '--quiet=true'] + jfrog_args
+    subprocess.run(args, check=True)
```

Comment:
Why is deleting the folder necessary?  Is it because the number and names of the new files might be different?

Reply:
Exactly. Any rule change, or even other code changes, can produce a completely new set of files, sometimes more, sometimes less, files with the same name but different contents, or the same contents but in different names, etc.

It is easier to remove the folder completely than keep track of files. The file-based operations, currently, do not add missing files nor does it delete truth files if non-existent. This may be a feature to add, but not in this PR. The intent is exactly this: If anything is different in a folder, remove and rebuild the folder.